src/services/sortingSchemes.js

```diff
@@ -27,6 +27,20 @@ export const trickSortingSchemes = [
     "attributeFunc": (a) => a.stickFrequency,
     "showCategory": false,
   },
+  {"name": "Starting Position",
+  "id": 4,
+  "sortFunc": (a, b) => {return a.startPos.localeCompare(b.startPos, undefined, {numeric: true, sensitivity: 'base'});},
```

Comment:
I don't see the need for the options, the position are defined in an enum and therefore won't vary in spelling or punctuation